config/rbac/role.yaml

```diff
@@ -17,6 +17,7 @@ rules:
   resources:
   - events
   verbs:
+  - '*'
```

Comment:
remove create and patch if you set `*`

Reply:
yep, this makes sense, but why do we need `*` in the first place? IOW, why `create` and `patch` are not sufficient?

Reply:
we can revert, no biggy